ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/SandboxOnXRunner.scala

```diff
@@ -99,7 +100,9 @@ object SandboxOnXRunner {
         bridgeConfig = bridgeConfig,
         materializer = materializer,
         loggingContext = loggingContext,
-        metricsFactory = metrics.defaultMetricsFactory,
+        metricsFactory = {
+          metrics.defaultMetricsFactory: @nowarn
```

Comment:
this is an odd construction 😄 You ascribe this to the annotation type? or how does it work?

Reply:
I agree, but it seems this the only way to contain the `@nowarn` effect. It creates a local block, and this allow us to add the `@nowarn` to a specific line declaration.